apps/widgets-system/src/assets/img/arrow_down_black.svg

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="24px" height="24px" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
I think that it's a good idea if we create an "icons repository". To avoid icon duplication.. however I don't know if it's out of scope of our github code.

Reply:
I also think it's a good idea to have an icons repository, maybe separate from ui-toolkit. But first we need to check with the design team if there are any copyright issues we may not be aware, especially if we plan on making it a public repository.

The only reason those icons are being added here in this PR is so that they can be used in the demo.

Reply:
Oh, right! We should check for copyright issues right now, because we're adding some icons, and the ui-toolkit already is a public repository...

Reply:
please, check with them..